### PR TITLE
completes #106

### PR DESCRIPTION
**Pull request by BlakeDraper**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import { Component, OnInit } from '@angular/core';
...
 import { RegistrationComponent } from './registration/registration.component';
-import { CurrentUserService } from './services/current-user.service';
-import { AuthenticationService } from './services/authentication.service';
+import { CurrentUserService } from '@services/current-user.service';
+import { AuthenticationService } from '@services/authentication.service';
 import { MatDialog } from '@angular/material/dialog';
-import { StatesService } from './services/states.service';
-import { NetworkNamesService } from './services/network-names.service';
-import { SensorTypesService } from './services/sensor-types.service';
+import { StateService } from '@app/services/state.service';
```
Comment (lprivette) [inline src/app/app.component.ts:8]: why is this one "@app/services/state.service" unlike the other ones that are "@services/blah.service"?

Comment (BlakeDraper) [reply src/app/app.component.ts:8]: because of auto-import not being aware of the TS config shortcuts we have. Will fix.

Comment (BlakeDraper) [reply src/app/app.component.ts]: fixed

File: src/app/app.module.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ import { RegistrationComponent } from './registration/registration.component';
 import { ConfirmComponent } from './confirm/confirm.component';
 import { CurrentUserService } from '@services/current-user.service';
 import { AuthenticationService } from '@services/authentication.service';
-import { EventsService } from '@services/events.service';
+import { EventService } from '@app/services/event.service';
```
Comment (lprivette) [inline src/app/app.module.ts:43]:
```suggestion
import { EventService } from '@services/event.service';
```

**Review by lprivette — APPROVED** [review]

**Pull request merged**